src/components/tab-bar/tab-bar.tsx

```diff
@@ -0,0 +1,155 @@
+import {
+    Component,
+    h,
+    Prop,
+    Element,
+    State,
+    EventEmitter,
+    Event,
+} from '@stencil/core';
+import { MDCTabBar, MDCTabBarActivatedEvent } from '@limetech/mdc-tab-bar';
+import { strings } from '@limetech/mdc-tab-bar/constants';
+import { Tab } from './tab.types';
+import config from '../../global/config';
+
+const { TAB_ACTIVATED_EVENT } = strings;
+
+@Component({
+    tag: 'limel-tab-bar',
+    styleUrl: 'tab-bar.scss',
+    shadow: true,
+})
+export class TabBar {
+    @Element()
+    private element: HTMLElement;
+
+    /**
+     * List of tabs to display
+     */
+    @Prop()
+    public tabs: Tab[];
+
+    private mdcTabBar: MDCTabBar;
+
+    @Event()
+    private changeTab: EventEmitter<Tab>;
+
+    @State()
+    private selectedTab = this.tabs[0];
```

Comment:
I'm not sure we need this variable. We should be able to get the old selected tab in `handleTabActivated` by looping through the given `this.tabs` and finding the one with `active === true`. Having two variables for the same thing makes it harder to keep them in sync